_posts/2019-11-19-apple-tv-plus.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title: Apple TV Plus
+date: 2019-11-19 16:28:05.000000000 +01:00
+type: post
+published: true
+status: publish
+categories: [Tech]
+image:
+image2:
+author: Valentino Urbano
+---
+
+I've finally tried Apple TV Plus for a few weeks and I can say that the shows are good, but the software is terrible.
+
+I'm only interested in Saw and For All Mankind so I did not watch anything else, but both are great and well worth a watch if you like the premises of the series.
+
+Apple strength should be in the software, but apparently it is not. We have seen something similar with the first version of Apple Music, the UI looked good, but it was not usable.
+
+I would expect a new app to be far behind the competition when it came to features, but here even the basics are still missing. There is no way to continue a show or if there is (so I'm told) it doesn't work most of the times. The UI looks good, but it is unusable. Apple TV Plus has now a few shows and it is already hard to navigate and find the exact show you want to watch, imagine how it would be if it has even 50 shows, let alone the number its competitors have.
```

Comment:
50 - > more

Reply:
Say you watch on your laptop, not in the app

Mention the buggy subtitles 

Say it's tedious to CLICK on the skip few sec button instead of using arrows and WHY there's no skip intro button?